src/components/RichTextField.tsx

```diff
@@ -55,6 +55,10 @@ export function RichTextField(props: RichTextFieldProps) {
   // Use a ref for onChange b/c so trixChange always has the latest
   const onChangeRef = useRef<RichTextFieldProps["onChange"]>(onChange);
   onChangeRef.current = onChange;
+  const onBlurRef = useRef<RichTextFieldProps["onBlur"]>(onBlur);
+  onBlurRef.current = onBlur;
+  const onFocusRef = useRef<RichTextFieldProps["onFocus"]>(onFocus);
+  onFocusRef.current = onFocus;
```

Comment:
We only hook the `trix-change` / `trix-blur` / `trix-focus` handlers once in a `useEffect`, because we don't control the lifetime of the trix JS object, which is fine, but means we need to use Refs to ensure we're always calling the latest callback that the user provided.